extensions/ql-vscode/src/vscode-tests/cli-integration/query.test.ts

```diff
@@ -151,7 +151,8 @@ describe('using the query server', function() {
             localChecking: false,
             noComputeGetUrl: false,
             noComputeToString: false,
-            computeDefaultStrings: true
+            computeDefaultStrings: true,
+            emitDebugInfo: true
```

Comment:
Future note: we may want to parameterise/matrix this test so that we check both `true/false` for `emitDebugInfo`.